packages/manager/apps/key-management-service/src/data/api/identity.ts

```diff
@@ -0,0 +1,87 @@
+import apiClient, { ApiResponse } from '@ovh-ux/manager-core-api';
+import {
+  Identity,
+  IdentityGroup,
+  IdentityOauthClient,
+} from '@/types/identity.type';
+
+/**
+ *  Retrieve all IAM users of this account
+ */
+
+export const getIdentityUsersIds = async (): Promise<ApiResponse<string[]>> => {
+  return apiClient.v6.get(`me/identity/user`);
```

Comment:
**nitpick** : make (exportable) constants for routes (like `me/identity/user`)

Reply:
Well it's not that useful to create a constant just to avoid typing the api route twice in a file

Reply:
But maybe you should use `useResourceV6` instead of the get directly